src/api/sarifViewerApi.ts

```diff
@@ -53,9 +46,7 @@ export interface Api {
     closeAllLogs(): Promise<void>;
 
     /**
-     * A set of root paths to use for mapping remote artifact locations.
-     * The paths are used only for this instance of the extension.
-     * They are not persisted in the user settings.
+     * A set of base URIs to use for mapping remote artifact locations.
      */
-    rootPaths: vscode.Uri[];
+    baseUris: ReadonlyArray<vscode.Uri>;
```

Comment:
I was off in my suggestion. I just checked the SARIF spec and they refer it to as `uriBase` and `uriBases`.

Reply:
Fixed. Thanks for catching this before I completed the PR :)

---
In reply to: [418274205](https://github.com/microsoft/sarif-vscode-extension/pull/276#discussion_r418274205) [](ancestors = 418274205)